src/afl-fuzz-one.c

```diff
@@ -364,7 +364,7 @@ static void locate_diffs(u8 *ptr1, u8 *ptr2, u32 len, s32 *first, s32 *last) {
 
 #endif                                                     /* !IGNORE_FINDS */
 
-#define BUF_PARAMS(name) (void **)&afl->name##_buf, &afl->name##_size
+#define BUF_PARAMS(name) (void **)&afl->name##_buf
```

Comment:
if we would call this AFL_BUF_PARAM we could put it in types.h and never would need to define + undefine it.

Reply:
and the S at the end can be removed anyway as it not only the buf param, and not also the size